src/ralph/export_to_ng/publishers.py

```diff
@@ -7,13 +7,23 @@
 from django.dispatch import receiver
 from pyhermes import publisher
 
-from ralph.business.models import Venture, VentureRole
+from ralph.business.models import (
+    RoleProperty,
+    RolePropertyValue,
+    Venture,
+    VentureRole
+)
 from ralph.discovery.models import Device
 
 logger = logging.getLogger(__name__)
 
 
-def ralph3_sync(model):
+ROLE_PROPERTY_WHITELIST = getattr(
```

Comment:
why not just put this to the settings with default value? we should read it from env variable either way